### ISSUE TITLE
limit hysteresis would be helpful

### ISSUE THREAD

**Issue by Kaiman77** [actuallymentor/battery#222]
**Is your feature request related to a problem? Please describe.**
When the battery reachd the 80% limit, everything is fine until it drops to 79% for whatever reason.
Then the tool triggers the charging up to 80% again until the battery drops to 79% again.

**Describe the solution you'd like**
Adding the possibility for a hysteresis would be helpful.
Basically saying, charge up to 80% only if the battery drops below 70% (or whatever value).

**Describe alternatives you've considered**
I don't see an alternative, that's a pretty typical way to do it in automation control

**Comment by actuallymentor** [comment]
Cool idea. Do you have the time/bandwidth to draft a PR?

**Comment by Kaiman77** [comment]
time/bandwidth yes, but my skill level doesn't allow me to, sorry ;-)

**Comment by actuallymentor** [comment]
Alas, I will tag it as a feature request and hope that someone with more time than me can pick it up :)

**Comment by earthsound** [comment]
@actuallymentor dupe of issue #150?

**Comment by seefood** [comment]
Yup, seems to me like a clone of #150, which includes some decent bash and applescript workarounds. @Kaiman77 I suggest closing.

**Comment by Kaiman77** [comment]
I agree to close as duplicate of #150
